cookbook-react/recipes/inputs/barcelona/index.test.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { render, queryByAttribute }  from '@testing-library/react';
+
+import Input from '.';
+
+jest.mock('i18next', () => ({
+  t: (key: string) => key
+}));
+
+describe('Barcelona Input', () => {
+  const staticProps = { label: 'My Label', placeholder: 'Some Placeholder', name: 'email'  }
+
+  describe('when input is confirmable and there is no error', () => {
+    const component = <Input {...staticProps} />;
+
+    it('shows a valid input content', () => {
+      const instance = TestRenderer.create(component)
+      expect(instance.toJSON()!).toMatchSnapshot();
+    })
+
+    it('does not show the error message', async () => {
+      const { getByRole } = render(component);
+      const errorElement = await getByRole('alert');
+      expect(errorElement?.classList).not.toContain('show');
```

Comment:
Maybe you could use `.not.toBeVisible()`, because checking classes is too related to the implementation. What do you think?

Reply:
Wasn't able to make it work because tests don't load css. Changed it to toHaveclass